docs/sources/operations/scalability.md

```diff
@@ -27,3 +27,63 @@ The query scheduler process itself can be started via the `-target=query-schedul
 In compute-constrained environments, garbage collection can become a significant performance factor. Frequently-run garbage collection interferes with running the application by using CPU resources. The use of memory ballast can mitigate the issue. Memory ballast allocates extra, but unused virtual memory in order to inflate the quantity of live heap space. Garbage collection is triggered by the growth of heap space usage. The inflated quantity of heap space reduces the perceived growth, so garbage collection occurs less frequently.
 
 Configure memory ballast using the ballast_bytes configuration option.
+
+## Remote Rule Evaluation
+
+This feature was introduced as part of [`LID-0002`](https://github.com/grafana/loki/pull/8129).
```

Comment:
I don't think this line is necessary.  
It is also incorrect, the LID is just the design doc and discussion, it doesn't change the code or introduce new features.

Reply:
You're right, I could've worded it better; I want to include the design doc so folks can read up on the design considerations if they want to.

How's this?

```suggestion
This feature was initial proposed in [`LID-0002`](https://github.com/grafana/loki/pull/8129).
```